driver-core/src/test/java/com/datastax/driver/core/CCMBridge.java

```diff
@@ -40,7 +40,11 @@
 
     private static final Logger logger = LoggerFactory.getLogger(CCMBridge.class);
 
-    private static final String CASSANDRA_VERSION;
+    private static final String INPUT_CASSANDRA_VERSION;
+
+    private static final VersionNumber CASSANDRA_VERSION_NUMBER;
```

Comment:
How about calling these constants `GLOBAL_XXX` to match their getter method names?